packages/lib-webpack/src/webpack/PatchContainerEntryPlugin.ts

```diff
@@ -0,0 +1,38 @@
+import { remoteEntryScript, remoteEntryCallback } from '@openshift/dynamic-plugin-sdk';
+import * as webpack from 'webpack';
+
+export class PatchContainerEntryPlugin implements webpack.WebpackPluginInstance {
+  constructor(private readonly pluginName: string) {}
```

Comment:
No need to call `super()` here either?

I'm getting a lint error on both of these for "useless constructor" but that's not true because of the property creation by TS, maybe our linter rules need to be updated?

Reply:
The ESLint "useless constructor" means constructor with empty body.

This is **not** adequate for TypeScript constructors which use short-hand syntax to initialize class properties.

Above code is short-hand equivalent to
```ts
export class PatchContainerEntryPlugin implements webpack.WebpackPluginInstance {
  private readonly pluginName;

  constructor(pluginName: string) {
    this.pluginName = pluginName;
  }

  // .. rest of code
}
```

Reply:
Reference: https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties

Reply:
No I understand that... but my IDE is showing me the lint errors you have just not allowed to be run by the npm script. We'll need to fix it in the lint issue https://issues.redhat.com/browse/HAC-761